plugins/frontend/backstage-plugin-travis-ci/src/api/index.ts

```diff
@@ -96,7 +96,7 @@ export class TravisCIApiClient implements TravisCIApi {
   }
 
   async retry(buildNumber: number): Promise<Response> {
-    const idToken = await this.identityApi.getIdToken();
+    const idToken = await this.identityApi.getCredentials();
```

Comment:
```suggestion
    const { token } = await this.identityApi.getCredentials();
```

Reply:
The getCredentials() should return an object now with a token property, we'll need to put that token into the header